.github/workflows/unittests.yml

```diff
@@ -56,12 +56,12 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python: [2.7, pypy2, pypy3, 3.5, 3.6, 3.7, 3.8]
+        python: [2.7, pypy2, pypy3, 3.5, 3.6, 3.7, 3.8, 3.9]
```

Comment:
What do you think about removing 3.5 from here? It also went EOL but unlike 2.7 it's barely used.

Reply:
I will rather keep it unless there is a feature that we cannot implement.